include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -554,8 +554,8 @@ inline bool constexpr all_satisfy_aligned_seq<type_list<elems...>> = (aligned_se
  */
 template <typename tuple_t, typename char_t>
 //!\cond
-    requires !std::ranges::input_range<tuple_t> &&
-             !alphabet<tuple_t> && // exclude alphabet_tuple_base
+    requires (!std::ranges::input_range<tuple_t>) &&
+             (!alphabet<tuple_t>) && // exclude alphabet_tuple_base
              tuple_like<remove_cvref_t<tuple_t>> &&
```

Comment:
```suggestion
              tuple_like<remove_cvref_t<tuple_t>> &&
```

Reply:
Sorry about that multiple single line suggestions. I noticed multi-line suggestions later on.

Reply:
This has correct indentation, because our general rule is to align them on the same expression.

```c++
requires |expression1
         |expression2
         |expression3
```

I don't know if you got confused, because the first expression is `(!alphabet<tuple_t>)` is enclosed in parenthesis.

You would need to add spaces if you span more expression within a pair of parenthesis.

Like so:

```c++

requires |(expression1
         | expression2)
         |expression3
```

I will mark all comments with :rocket: which I think don't have this indentation issue.

Reply:
Oh, you are right.